introduction.md

```diff
@@ -0,0 +1,3 @@
+This is a first file prepared in the git during Mastering Github by Malay Patel
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.